rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -612,6 +612,8 @@ module PatternTrees {
       result = rank[i + 1](Pat pat, int j | pat = this.getPat(j) | pat order by j)
     }
 
+    predicate isEmpty() { not any(Pat p) = this.getPat(0) }
```

Comment:
`not exists(this.getPat(_))`.

Reply:
True. I realized that that was a bug, fixed in a later commit and after https://github.com/github/codeql/pull/17800/commits/6568eb80a2bbe77beb044af52afa13bb96b55208 it got removed as the behavior is now inherited from `StandardTree`.